osidb/models.py

```diff
@@ -1,6 +1,7 @@
 """
 draft model for end to end testing
 """
+
```

Comment:
I wonder why you add the new line here. Is it based on some linter? Anyway I see I have not read the module description for very long probably :smile: and it is totally outdated - most probably from the very beginning of the project. When touching this piece you can remove the whole comment.

Reply:
Apparently the newline comes from my local version of `black` (24.1.1) which differs from the one in the pre-commit hooks (22.12.0). The new version [enforces this newline](https://black.readthedocs.io/en/latest/change_log.html#id10), which is why my editor must've added it, whereas the old version doesn't care.

I will remove this docstring anyway as it seems pretty useless.